zingolib/src/lightclient/propose.rs

```diff
@@ -132,12 +130,13 @@ impl LightClient {
     /// Unstable function to expose the zip317 interface for development
     // TOdo: add correct functionality and doc comments / tests
     // TODO: Add migrate_sapling_to_orchard argument
+    #[cfg(test)]
     pub async fn propose_send_all(
         &self,
-        _address: Address,
-        _memo: Option<MemoBytes>,
+        _address: zcash_keys::address::Address,
+        _memo: Option<zcash_primitives::memo::MemoBytes>,
```

Comment:
I intentionally wrote this function with implicit imports because it looks much better to me, i dont like code that looks unnecesarrily complex or convoluted. my opinion is that we shouldnt have our own Address or MemoBytes structs, we should use LRZ's types, and should be very familiar with LRZ types, so when we see `Address` it is already ingrained this is an LRZ address type. I think the best way for our split opinions on this to live together is i wont change your functions and you dont change mine haha

Reply:
I see your point.  I don't remember with certainty why I made this change, but I suspect it was that I was moving the function and didn't want to add-and-delete "`use FOO`" in **addition** to moving the fn itself.

I agree with your assertion that we should use LRZ types wherever possible.